app/views/vacancies/index.html+phone.haml

```diff
@@ -9,7 +9,7 @@
 
   .govuk-grid-column-two-thirds
     .govuk-grid-column-full.vacancies-count{class: @vacancies.user_search? ? 'search-made' : ''}
-      %p.govuk-heading-l.mb0= @vacancies.total_count_message
+      %p.govuk-heading-l.mb0= @filters.location_category_search? ? @vacancies.total_count_message_with_location(@filters.location) : @vacancies.total_count_message
```

Comment:
Is it possible to move this logic into `VacanciesPresenter` ?

Reply:
Not sure. Since `@filters` is available to the view, it was easier to pass in the `location name` and also check if a `location category search` was performed from within the view.

I did try to put all the logic in the presenter but my first attempt had me trying to make the `location name` and `location_category_seach?` values accessible by the `Vacancy Presenter` (which meant passing them in from another class as variables) and then changing everywhere that `Vacancy Presenter` was called.

**I'm open to any suggestions.**

Reply:
Yep, looks like you would need to initialise the `VacancyPresenter` with the `filters` to be able to do that, not sure which route is better. I think both solutions are acceptable